### PR TITLE
[Enhancement] Add read tablet ability to PseudoCluster

### PR DESCRIPTION
**Pull request by decster**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] refactor
- [ ] others

## Problem Summary(Required) ：

This PR adds tablet read logic to PseudoCluster

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: fe/fe-core/src/test/java/com/starrocks/pseudocluster/PseudoBackend.java [hunk 1]
```diff
@@ -831,6 +862,11 @@ void execBatchPlanFragment(TExecPlanFragmentParams commonParams, TExecPlanFragme
         report.setDone(true);
         // TODO: support error injection
         report.setStatus(new TStatus(TStatusCode.OK));
+        for (TPlanNode planNode : commonParams.fragment.plan.nodes) {
+            if (planNode.node_type == TPlanNodeType.OLAP_SCAN_NODE) {
```
Comment (chaoyli) [inline fe/fe-core/src/test/java/com/starrocks/pseudocluster/PseudoBackend.java:866]: Does the new pipeline use another ScanOperator, not the OLAP_SCAN_NODE?

Comment (decster) [reply fe/fe-core/src/test/java/com/starrocks/pseudocluster/PseudoBackend.java:866]: I tested it, it works fine, so it should be OLAP_SCAN_NODE

Comment (chaoyli) [reply fe/fe-core/src/test/java/com/starrocks/pseudocluster/PseudoBackend.java:866]: Maybe in the FE still use the OLAP_SCAN_NODE?
In BE, it have changed to exec/pipeline/scan/olap_scan_operator.cpp.

Comment (decster) [reply fe/fe-core/src/test/java/com/starrocks/pseudocluster/PseudoBackend.java:866]: That doesn't matter I think

**Review by nshangyiming — APPROVED** [review]
LGTM

**Review by chaoyli — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Review by Astralidea — APPROVED** [review]

**Pull request merged**